.github/workflows/code_scan.yml

```diff
@@ -7,6 +7,26 @@ on:
     - cron: "0 18 * * 1-5"
 
 jobs:
+  Bandit:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
It couldn't matter, but why using actions/checkout@v2 instead of actions/checkout@v3?

Reply:
@eunwoosh  Thanks for your finding!